src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs

```diff
@@ -56,6 +55,8 @@ public class ApplicationController : Controller
             _shellDescriptor = shellDescriptor;
         }
 
+        public dynamic New { get; }
```

Comment:
ShapeFactory? Or is it consistent with what we doing by using New?

Reply:
I've seen this pattern very often in the code.

Reply:
Ya

Reply:
Maybe inspired of our `RazorPage.New` property often used in razor views templates

        public dynamic New
        {
            get
            {
                return Factory;
            }
        }

Reply:
Do we want to use a different approach?
Or can we accept this PR? :)

Reply:
IMHO what's done now is fine, so :shipit:

Reply:
Then I will resolve

Reply:
It's fine ;)